File: CCAddCounter/CCAddCounter/ViewController.m
//
//	ViewController.m
//		CCAddCounter
//		Chen Chen @ Apirl 12th, 2015
//

#import "ViewController.h"

@interface ViewController ()

/// 加数文本框
@property (weak, nonatomic) IBOutlet UITextField *numberA;
/// 被加数文本框
@property (weak, nonatomic) IBOutlet UITextField *numberB;
/// 加法和标签
@property (weak, nonatomic) IBOutlet UILabel *sumAnswer;

@end

@implementation ViewController

#pragma mark - 系统方法

/**
 *  视图已经加载方法
 */
- (void)viewDidLoad {
    
    [super viewDidLoad];
}

#pragma mark - 加法方法

/**
 *  加法计算方法
 */
- (IBAction)addCounter {
    
    NSInteger a = [self.numberA.text integerValue];
    NSInteger b = [self.numberB.text integerValue];
    self.sumAnswer.text = [NSString stringWithFormat:@"%ld", a + b];
    
    [self.view endEditing:YES];
}

@end

File: CCAddCounter-Swift/CCAddCounter-Swift/ViewController.swift
//
//	ViewController.swift
//		CCAddCounter
//		Chen Chen @ August 15th, 2016
//

import UIKit

class ViewController: UIViewController {

    /// 加法计算视图
    var addCounterView = AddCounterView(frame: kScreenFrame)

    // MARK: - 系统方法

    /**
     视图已经加载方法
     */
    override func viewDidLoad() {

        super.viewDidLoad()

        view = addCounterView
        addCounterView.addButton.addTarget(self, action: #selector(addCounter), for: .touchUpInside)
    }

    // MARK: - 界面方法

    /**
     加法计算方法
     */
    @objc fileprivate func addCounter() {

        let a: Int = Int(addCounterView.numberA.text!) ?? 0
        let b: Int = Int(addCounterView.numberB.text!) ?? 0
        addCounterView.sumAnswer.text = String(a + b)

        view.endEditing(true)
    }

}
